iot_verifier/src/reward_share.rs

```diff
@@ -898,14 +934,22 @@ mod test {
             reward_shares_in_dec(dec!(150), dec!(350), gw6_dc_spend),
         ); // 0.0150, 0.0350
 
-        let gw_shares = GatewayShares { shares };
+        let mut gw_shares = GatewayShares {
```

Comment:
Instead of doing this, construct a new set of gateway shares from an iterator of hashmaps

Reply:
Addressed in sep PR via this commit: d1238dc8e5e255b4aa03f3c4a41fc6da62053b04